object/permission_enforcer.go

```diff
@@ -284,24 +284,40 @@ func removeGroupingPolicies(permission *Permission) error {
 	return nil
 }
 
-type CasbinRequest = []interface{}
+// type CasbinRequest = []interface{}
```

Comment:
type CasbinRequest = []string

Reply:
@xiao-kong-long then remove all of it, and don't just comment it, remove

![image](https://github.com/casbin/casdoor/assets/3787410/d9763b34-cdd2-4ef7-bc99-5a0084aa8823)